platform/master/inst/tests/test_3_foreach.R

```diff
@@ -159,3 +159,12 @@ test_that("Dense darrays: works", {
 
   })
 
+test_that("testing deterministic behavior of foreach", {
```

Comment:
We are moving away fro RUnit. Its possible that RUnit is removed from installation script and replaced with testthat. Better to use some other library for the unit test.